src/reducers/HelpDropdownState.ts

```diff
@@ -0,0 +1,24 @@
+import { StatusState } from '../store/Store';
+import { HelpDropdownActionKeys } from '../actions/HelpDropdownActions';
+
+const INITIAL_STATE: StatusState = {
+  status: {},
+  products: [],
+  warningMessages: []
+};
+
+// This Reducer allows changes to the 'serviceGraphDataState' portion of Redux Store
+const HelpDropdownState = (state: StatusState = INITIAL_STATE, action) => {
```

Comment:
Missing type of action ?

Reply:
Not, If you see another reducers there is not type in action because you can store any kind of object.